backend/endpoints/config.py

```diff
@@ -115,25 +115,44 @@ async def delete_platform_version(request: Request, fs_slug: str) -> MessageResp
     return {"msg": f"{fs_slug} version removed successfully!"}
 
 
-# @protected_route(router.post, "/config/exclude", ["platforms.write"])
-# async def add_exclusion(request: Request) -> MessageResponse:
-#     """Add platform binding to the configuration"""
+@protected_route(router.post, "/config/exclude", ["platforms.write"])
```

Comment:
With #1247 now merged, we can use Scope objects for these two routes!

Reply:
True! let me adapt it